avocado/utils/ssh.py

```diff
@@ -214,3 +214,28 @@ def quit(self):
         :rtype: bool
         """
         return self._master_command('exit')
+
+    def copy_files(self, source, destination, recursive=False):
+        """
+        Copy Files to and from remote through scp session
+        :param source: Local file
```

Comment:
I think that "local" is not accurate since I can copy from a remote host to my local machine with this logic, right?